### ISSUE TITLE
Suggestion: Reorganization of the CO Selection screen

### ISSUE THREAD

**Issue by Seimix** [Robosturm/Commander_Wars#73]
Now yes, I know this is a minor issue... It's just that it has always bothered me that the factions were ordered based on the first letter of their first COs.
(I.E Brown Desert is the first faction that shows up when you're choosing a CO because its first CO is "Adam" and Black Hole is the second faction because their first CO is Adder, and so on) 
So I figured the order of the factions should be fixed. The factions order should be (coming from left to right) :
^ = Can be changed

Orange Star > Blue Moon > Green Earth > Yellow Comet > Black Hole > Bolt Guard > Amber Corona > Brown Desert > Teal Islands > Dark Matter > Metall Army^ > Pink Frontier^ > Golden Sun^

The OC factions were a bit harder to organize... but I organized them this way because I'm assuming Metall Army is a CoW only thing and PF comes before GS in the current way it's ordered. Otherwise, afaik they can be organized in any way. (I'm not familiar with Custom Wars at all in any shape or form, I didn't even know about its existence until I found this fangame)
If possible, any new factions modded in should be placed after GS automatically.

As for the organization of the COs in the factions...is a bit more complicated. The COs being organized alphabetically imo is fine, although things I would definitely change if I could reorganize the COs, would be:

*Have the blank faction COs always appear first in whatever faction it's in
*Have the Random CO be the last selectable CO under whichever faction it's in.

Now, the COs themselves (not counting blanks or random) can probably be organized better, as I have an idea as to how they could be reorganized, but I'm not really going to be going into that rn as I don't care much of the way they are currently organized.

**TL;DR** this is basically what I would like changed in one image: (Ignore the black background)
https://i.imgur.com/7dp1Bwk.png

**Comment by Robosturm** [comment]
Reworked.
You can change the army and co-order if you want in the next release.
Made it moddable
